lib/ansible/parsing/vault/__init__.py

```diff
@@ -319,10 +319,9 @@ def bytes(self):
         return None
 
     def load(self):
-        self._bytes = self.read_file(self.filename, self.loader)
+        self._bytes = self.read_file(self.filename)
 
-    @staticmethod
-    def read_file(filename, loader):
+    def read_file(self, filename):
```

Comment:
Ah I see... but later you want to make this overridable by subclasses....  subclasses should be able to override a staticmethod with a regular method without difficulty so I think this should remain a method but also remain a staticmethod.  Depending on what the pylint warning is you can disable it here with a comment that this is a method because the method is overridden by subclasses.